github-issue-classification/scripts/get-data.sh

```diff
@@ -36,7 +36,7 @@ get_data () {
   do
     URL+="?per_page=100&page="
     #curl -H "Authorization: token $TOKEN" "$URL$PAGE" > $DLDIR/issues-$((100 * PAGE)).json
-    curl -H "$AUTH" "$URL$PAGE" > $DLDIR/issues-$((100 * PAGE)).json
+    curl "$URL$PAGE" > $DLDIR/issues-$((100 * PAGE)).json
```

Comment:
@bd-dean authorization header removed? this was added to handle github api rate limit error for unauthenticated access.

Reply:
If you use the -H flag and dont give an argument so that $AUTH is effectively empty, curl fails. The -H flag fails on two different Clear Linux systems when set up like this.   I had to set --user "user:password" in the script to get around api rate limit, no matter what value I set for $AUTH.

Reply:
@mak-454 does the comment from @bd-dean make sense, if so can we merge this?

Reply:
@bd-dean @unrahul apologies for delayed response.
I just ran the script as is inside the clearlinux container and it worked. I used the docker, **clearlinux/stacks-dars-mkl**. I have copied the download logs at the end.
Could you please paste the command you tried and failed? 
Token is better than username & password to get around the API rate limit issue, because tokens can be revoked after the purpose is complete.


root@ddb6eeceee1e/workdir/scripts # ./get-data.sh --dir /tmp/data/
==============================================================
Wed 30 Oct 2019 10:09:07 AM UTC -- 
==============================================================
Get github issues data, if no <issues_url> provided, issues from ClearLinux github will be downloaded.
==============================================================
Wed 30 Oct 2019 10:09:07 AM UTC -- get issues data from ::  https://api.github.com/repos/clearlinux/distribution/issues
==============================================================
Loading required manifests...
Warning: Bundle "jq" is already installed, skipping it...
1 bundle was already installed
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     33      0 --:--:-- --:--:-- --:--:--    34
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     53      0 --:--:-- --:--:-- --:--:--    53
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     56      0 --:--:-- --:--:-- --:--:--    56
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     45      0 --:--:-- --:--:-- --:--:--    45
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     50      0 --:--:-- --:--:-- --:--:--    50
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     11      0 --:--:-- --:--:-- --:--:--    11
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     43      0 --:--:-- --:--:-- --:--:--    44
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     52      0 --:--:-- --:--:-- --:--:--    52
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     53      0 --:--:-- --:--:-- --:--:--    53
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     48      0 --:--:-- --:--:-- --:--:--    48
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     42      0 --:--:-- --:--:-- --:--:--    42
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     14      0 --:--:-- --:--:-- --:--:--    14
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     18      0 --:--:-- --:--:-- --:--:--    18
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     49      0 --:--:-- --:--:-- --:--:--    49
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     5  100     5    0     0     44      0 --:--:-- --:--:-- --:--:--    44
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  219k  100  219k    0     0   228k      0 --:--:-- --:--:-- --:--:--  228k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  439k  100  439k    0     0   262k      0  0:00:01  0:00:01 --:--:--  262k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  413k  100  413k    0     0   331k      0  0:00:01  0:00:01 --:--:--  330k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  472k  100  472k    0     0   338k      0  0:00:01  0:00:01 --:--:--  338k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  408k  100  408k    0     0   327k      0  0:00:01  0:00:01 --:--:--  327k
==============================================================
Wed 30 Oct 2019 10:09:16 AM UTC -- Done in :  0 minute(s)
==============================================================

Reply:
Reverted to add -H back to command